woocommerce/woocommerce
# Shipping Settings: Add busy animation to Continue button

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/40740

This PR adds animation to the "Continue" button instead of the spinner gif that didn't match the new look of the modals.

<img width="551" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/6d64d3e7-7bf9-4461-bfa2-ed3c8d0bb200">



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new Shipping Zone, `/wp-admin/admin.php?page=wc-settings&tab=shipping&zone_id=new`
2. Add a new Shipping method. Note the primary button styles during the ajax request when continuing to the configure screen.
3. Alternatively, add the `is-busy` class to the "Continue" button when adding a new method to see the styles for longer than a brief second.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update Shipping Settings zone method modal spinner on modal primary button.

#### Comment 

</details>
